Sources/WebUI/WebViewProxy.swift

```diff
@@ -186,4 +189,15 @@ public final class WebViewProxy: ObservableObject {
     public func clearAll() {
         webView?.remake()
     }
+
+    /// A container for capturing the web view's content.
+    public struct ContentReader {
+        let wkWebView: WKWebView
```

Comment:
Is it okay to hold a strong reference to the WKWebView instance? I'm concerned about potential memory leaks.

Reply:
After checking for memory leaks using the same method as in #36, I confirmed that the `WKWebView` was properly deinitialized after onDismiss.